crypto/test/cipher_driver.c

```diff
@@ -467,7 +467,7 @@ srtp_err_status_t cipher_array_alloc_init(srtp_cipher_t ***ca,
     /* allocate key */
     key = srtp_crypto_alloc(klen_pad);
     if (key == NULL) {
-        free(cipher_array);
+        srtp_crypto_free(cipher_array);
```

Comment:
cipher_array is allocated with malloc not srtp_crypto_alloc, see 
` cipher_array =  (srtp_cipher_t **)malloc(sizeof(srtp_cipher_t *) * num_ciphers);`
so either that also needs to change or else go back to using free.